src/pages/feedback/general.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+  Text,
+  HStack,
+  useDisclosure,
+  AlertStatus,
+} from '@chakra-ui/react';
+import { FeedbackLayout, FeedbackAlert } from '@components';
+import { useStore } from '@store';
+import { Field, FieldProps, Form, Formik, FormikHelpers } from 'formik';
+import { NextPage } from 'next';
+import { useState } from 'react';
+
```

Comment:
All these pages should export that default getServerSideProps
```typescript
export { injectSessionGSSP as getServerSideProps } from '@ssrUtils';
```
unless they implement it.